src/response.rs

```diff
@@ -20,6 +21,19 @@ impl From<PeriodTime> for SystemTime {
     }
 }
 
+#[derive(Debug, PartialEq)]
+pub struct MetricsHierarchy(HashMap<String, Vec<String>>);
```

Comment:
I think we should have define the `Metric` type, but that affects other parts of the code, so I'd rather change that in a different PR.